src/index.html

```diff
@@ -17,11 +17,256 @@
+          <div class="card__price-value">$2,199</div>
+        </section>
+
+        <!--button buy-->
+        <button class="card__button-buy">BUY</button>
+      </div>
+      <div class="card">
+        <!--Product name and code-->
+        <div class="card__text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__prod-code">Product code: 195434</div>
+
+        <!--product reviews and stars avaliation-->
+        <section class="card__reviews">
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-text">Reviews: 5</div>
+        </section>
+
+        <!--product price-->
+        <section class="card__price">
+          <div>Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </section>
+
+        <!--button buy-->
+        <button class="card__button-buy">BUY</button>
+      </div>
+      <div class="card">
+        <!--Product name and code-->
+        <div class="card__text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__prod-code">Product code: 195434</div>
+
+        <!--product reviews and stars avaliation-->
+        <section class="card__reviews">
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-text">Reviews: 5</div>
+        </section>
+
+        <!--product price-->
+        <section class="card__price">
+          <div>Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </section>
+
+        <!--button buy-->
+        <button class="card__button-buy">BUY</button>
+      </div>
+      <div class="card">
+        <!--Product name and code-->
+        <div class="card__text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__prod-code">Product code: 195434</div>
+
+        <!--product reviews and stars avaliation-->
+        <section class="card__reviews">
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-star"></div>
+          <div class="card__reviews-text">Reviews: 5</div>
+        </section>
+
+        <!--product price-->
+        <section class="card__price">
+          <div>Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </section>
+
+        <!--button buy-->
+        <button class="card__button-buy">BUY</button>
```

Comment:
The product details, including name, code, reviews, and price, are repeated multiple times with the same content. If this is intentional for layout testing, it is fine. However, for a real application, ensure that each product card displays unique data.